snippets/conserv_with_d3.json

```diff
@@ -0,0 +1,3 @@
+{
+  "js": ["/node_modules/d3/d3.min.js"]
+}
```

Comment:
oh I am sorry - I didn't look close enough and thought you just use d3, btw

"buildJS": ["https://d3js.org/d3-scale.v0.7.min.js"]